sbaid/model/database/global_sqlite.py

```diff
@@ -0,0 +1,248 @@
+"""This module contains the GLobalSQLite class."""
+import sqlite3
+
+from gi.repository import GLib, Gio
+
+from sbaid.common.a_display import ADisplay
+from sbaid.common.b_display import BDisplay
+from sbaid.common.simulator_type import SimulatorType
+from sbaid.common.vehicle_type import VehicleType
+from sbaid.model.database.global_database import GlobalDatabase
+
+
+class GlobalSQLite(GlobalDatabase):
+    """This class implements the GlobalDatabase interface which allows for the all results
+    and project metadata to be stored."""
+    _connection: sqlite3.Connection
+
+    async def open(self, file: Gio.File) -> None:
+        """Creates a new database file and initialize the database schema."""
+        if not file.query_exists():
+            file.create_async(Gio.FileCreateFlags.NONE, 0)  # pylint: disable=no-member
+        else:
+            raise FileExistsError("File already exists")
```

Comment:
ich dachte open wird immer aufgerufen und soll das file halt laden wenn's schon existiert. weil wo kommst sonst die sqlite Connection her?

Reply:
Ja können wir so machen. Also einfach kein else?
```suggestion
           if not file.query_exists():
               file.create_async(Gio.FileCreateFlags.NONE, 0)  # pylint: disable=no-member
           path = file.get_path()
           ...
```

Reply:
ja aber wenn dus schon anders irgendwie gemacht hast können wirs auch so lassen?